### DISCUSSION TITLE
"Windows First Setup Screen" shows every time explorer is restarted / at login

### DISCUSSION THREAD

**Discussion by 0x01001100** [valinet/ExplorerPatcher#3192] [Q&A]
Basically, this screen is popping everytime I log at the system, and even when Explorer is restarted (so it was easy to take a screenshot of it)

![TheScreen](https://github.com/valinet/ExplorerPatcher/assets/86391643/d41c086c-e7f2-46d4-9a6e-ecb911fe6fec)

It clearly seems like it is the Screen that shows up when you first install an the OS, to configure some last settings like accessibility ones.. (IDK right now, but it is that last windows setup screen.... yall may know.. )

The system Specifications are:

> Windows 11 Pro
Version: 23H2
Compilation: 22631.3527
Windows Feature Experience Pack 1000.22700.1003.0

(Before April 23 update it was also happening; I saw the update today so I updated the system, and this still happens... )
Would appreciatte any comment about this. 
Thanks for everything! :)

**Comment by pyrates999** [comment]
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

The April 23rd update was a preview update.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

Since you installed the preview update, you need to install the latest pre-release of EP if you haven't yet: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.1_d11445a

**Reply by 0x01001100** [reply]
Thank for your reply. 
Yep, the ExplorerPatcher version is 22621.3527.65.1; 
![image](https://github.com/valinet/ExplorerPatcher/assets/86391643/cdbd5399-87c0-4820-a9e0-7a82f6112acd)


Maybe this helps more than uploading 10 screenshots of the settings as I saw at other posts; making a clean install of Explorer Patcher does not solve it..    settings seem  to not matter at all..
Im not using any other software that changes visuals on the system. 

The fact is, I recently lost a WD NVME disk (...it just died after 4 months of use, factory new........ ....Im trying not to talk about this. ....)
so I started using again an "old" SSD with a random -reallyunupdated- Windows 11 build on it. 
I reinstaled the whole Windows11 system on it, and when I applied ExplorerPatcher, this strange bug started to pop up at every boot, or sometimes when the ewindows explorer.exe is restarted....

Idk if "its just me"; It probably is this ***** windows.....
For now, thanks for your help to everyone around here. ^^

**Comment by pyrates999** [comment]
Let's check if there are file system errors then.  Please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.

**Comment by Amrsatrio** [comment]
Please check `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\OOBE\LaunchUserOOBE`. It must be 0.

**Reply by 0x01001100** [reply]
Hellothere,
There is no `LaunchUserOOBE` entry;
The reg.entry is `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\OOBE\SkipMachineOOBE`          
 (SkipMachineOOBE value: 1)

🙄 
as I stated bfore,
> Windows 11 Pro
Version: 23H2
Compilation: 22631.3527
Windows Feature Experience Pack 1000.22700.1003.0
